### PR TITLE
Dashboard - Quick strategy & Bot Builder (mobile)

### PR DESCRIPTION
**Pull request by paveetra-deriv**
Check to ensure that Bot Builder and Quick Strategy is accessible from Dashboard tab. 
Covering for Mobile & Responsive mode.

### CODE REVIEW

**Review by abuzar-deriv — APPROVED** [review]
LGTM

**Comment by abuzar-deriv** [comment]
Please don't use ci skip; we need the workflows.

File: cypress/e2e/smoke/dbot/DashboardQuickAction.cy.js [hunk 1]
```diff
@@ -1,24 +1,35 @@
 import BotDashboard from '../../../support/pageobjects/dbot/bot_dashboard_page'
 import QuickStrategy from '../../../support/pageobjects/dbot/quick_strategy'
+
 describe('QATEST-4128: Dashboard quick action to Quick Strategy and Bot Builder', () => {
```
Comment (meeyun-deriv) [inline cypress/e2e/smoke/dbot/DashboardQuickAction.cy.js:4]: this test id is outdated, please help to update it

**Review by meeyun-deriv — CHANGES_REQUESTED** [review]

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**